compare_mt/cache_utils.py

```diff
@@ -14,3 +17,12 @@ def return_cache_dict(key_list, value_list):
       raise ValueError(f'Only support caching for one system at a time!')
   cache_dict = {k:v[0] for (k, v) in zip(key_list, value_list)}
   return cache_dict
+
+class CachedPorterStemmer(PorterStemmer):
+  """A wrapper class for PorterStemmer that uses LRU cache to reduce latency"""
+  def __init__(self, mode=PorterStemmer.NLTK_EXTENSIONS):
+    super().__init__(mode)
+
+  @lru_cache(maxsize=6000)
```

Comment:
It seems that maxsize could be much bigger here? What about 50000?

Reply:
Sounds good! I increased the limit and it saves us an additional 14 seconds.